<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMediaPlayerClass</class>
 <widget class="QMainWindow" name="QtMediaPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtMediaPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsViewOrigin"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsViewAfter"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>500</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListView" name="fileList"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>檔案</string>
    </property>
    <addaction name="actionMenuOpenFile"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTBOpenFile"/>
   <addaction name="actionTBPlayVideo"/>
   <addaction name="actionPause"/>
  </widget>
  <action name="actionTBOpenFile">
   <property name="icon">
    <iconset resource="qtmediaplayer.qrc">
     <normaloff>:/QtMediaPlayer/icon/open.png</normaloff>:/QtMediaPlayer/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>開啟檔案</string>
   </property>
  </action>
  <action name="actionTBPlayVideo">
   <property name="icon">
    <iconset resource="qtmediaplayer.qrc">
     <normaloff>:/QtMediaPlayer/icon/play.png</normaloff>:/QtMediaPlayer/icon/play.png</iconset>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qtmediaplayer.qrc">
     <normaloff>:/QtMediaPlayer/icon/pause.png</normaloff>:/QtMediaPlayer/icon/pause.png</iconset>
   </property>
   <property name="text">
    <string>暫停</string>
   </property>
  </action>
  <action name="actionMenuOpenFile">
   <property name="text">
    <string>開啟檔案</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtmediaplayer.qrc"/>
 </resources>
 <connections/>
</ui>
